### PR TITLE
fix: local network hidden icon and custom connect card info

### PR DESCRIPTION
**Pull request by dandansad**

### CODE REVIEW

File: packages/eth-network/src/networkManager.js [hunk 1]
```diff
@@ -218,7 +218,6 @@ class NetworkManager {
     if (info && notify) {
       redux.dispatch('SELECT_NETWORK', `custom`)
       redux.dispatch('CHANGE_NETWORK_STATUS', true)
-      notification.success(`Network Connected`, `Connected to network at <b>${url}</b>`)
```
Comment (iNorthIsle) [inline packages/eth-network/src/networkManager.js:221]: why need to remove the notification

File: packages/eth-network/src/NetworkStatus/index.js [hunk 2]
```diff
@@ -26,8 +26,10 @@ export default function NetworkStatus(props) {
   return <>
     <UncontrolledButtonDropdown direction='up'>
       <DropdownToggle size='sm' color='default' className='rounded-0 px-2 text-muted'>
-        <span key={`network-${networkId}`} className={`${classnames('d-inline-block mr-1', connected ? 'color-success' : '')}`}>
-          <i className='fas fa-wifi mr-1' />
+        <span hidden={networkId == 'dev'}>
+          <span key={`network-${networkId}`} className={`${classnames('d-inline-block mr-1', connected ? 'color-success' : '')}`}>
+            <i className='fas fa-wifi mr-1' />
+          </span>
```
Comment (iNorthIsle) [inline packages/eth-network/src/NetworkStatus/index.js:32]:
```suggestion
          <span  hidden={networkId == 'dev'} key={`network-${networkId}`} className={`${classnames('d-inline-block mr-1', connected ? 'color-success' : '')}`}>
            <i className='fas fa-wifi mr-1' />
          </span>  
```

Comment (dandansad) [reply packages/eth-network/src/NetworkStatus/index.js:32]: d-inline-block classname will cause hidden failure

File: packages/eth-sdk/src/EthersClient/index.js [hunk 3]
```diff
@@ -207,11 +207,11 @@ class ExplorerProxy {
     }
 
     if (chainsHarmonyName.includes(this.networkId)) {
+      query.page = query.page - 1
```
Comment (iNorthIsle) [inline packages/eth-sdk/src/EthersClient/index.js:210]: how about 
```suggestion
      query.page -= 1
```
```suggestion
      query.page = query.page - 1
```

Comment (iNorthIsle) [inline packages/eth-sdk/src/EthersClient/index.js:210]: how about 
```suggestion
      query.page -= 1
```
```suggestion
      query.page = query.page - 1
```

**Review by iNorthIsle — APPROVED** [review]

**Pull request merged**